backend/main/src/Group/UseCases/AddFoodToContainerUseCase/AddFoodToContainerUseCase.ts

```diff
@@ -0,0 +1,96 @@
+import { IContainerRepository } from "src/Group/Domain/IContainerRepository";
+import { ContainerId } from "src/Group/Domain/Entities/Container";
+import { IUseCase } from "src/Shared";
+import { Err, Ok, Result } from "result-ts-type";
+import {
+	AddFoodToContainerUseCaseErrorType,
+	IAddFoodToContainerUseCase,
+	ContainerIsNotExisting,
+} from "src/Group/UseCases/AddFoodToContainerUseCase/IAddFoodToContainerUseCase";
+import { Food, FoodId, IFoodProps } from "src/Group/Domain/Entities/Food";
+import { Unit } from "src/Group/Domain/ValueObjects/Unit";
+import { Quantity } from "src/Group/Domain/ValueObjects/Quantity";
+import { Expiry } from "src/Group/Domain/ValueObjects/Expiry";
+
+/**
+ * Add a Food to a container.
+ */
+export class AddFoodToContainerUseCase
+	implements
+		IUseCase<
+			IAddFoodToContainerUseCase,
+			undefined,
+			AddFoodToContainerUseCaseErrorType
+		>
+{
+	private readonly containerRepository: IContainerRepository;
+
+	constructor(containerRepository: IContainerRepository) {
+		this.containerRepository = containerRepository;
+	}
+
+	public async execute(
```

Comment:
@hitohata 
Thank you for your comment! That's true. I don't have a concrete idea now, but I think that the authorization feature will be realized like this. Can you check if the idea seems to work?

To Get the User ID from HTTP header
- create Service to get UserId from CognitoAuthToken
- call the service, passing header of HTTP requests in the Router
- call the Controller, passing the user ID

To Verify User
- **add canEdit(userId) method to Group entity**
- **find Group related to the Container in this Usecase.**
- **call canEdit() in this Usecase.**

In this PR, I'll implement the feature shown in bold.

Reply:
@taniguchiiqqq 
Thank you for planning of implementation.
I think it would work. Let's go with that policy for now.

I also understand implementing the bold features in this PR. It sounds good.